### ISSUE TITLE
IE11-only bug: Cannot toggle columns after scrolling down column list

### ISSUE THREAD

**Issue by geotim90** [vaadin/framework#11245]
### Vaadin Framework version
* Originally discovered in **8.5.1**
* Reproducible in **8.5.2**

### Browser version
**Affects**:
* Internet Explorer 11.0.9600.19100CO with patch level 11.0.75
* Internet Explorer 11.0.6900.19101 with patch level 11.0.75

**Does not affect**:
* Mozilla Firefox ESR 52.9.0 (32-bit)
* Google Chrome 68.0.3440.106 (64-bit)

### Web container name and version
* Apache Tomcat 8.0.35
* Jetty (jetty-maven-plugin) 9.3.9.v20160517

### Description of the bug
When a Grid component has hidable columns, a list of columns is make available via an icon in the top right corner. If the list of hidable columns is too long to display in the available vertical space, the list is made scrollable. This works fine in Chrome and Firefox, however in IE11 it is impossible to select columns further down the list. Any time the user moves the mouse to select a list entry, the list instantly jump all the way to the top.

### Minimal reproducible example
**MyUI.java**
```java
public class MyUI extends UI
{
	@Override
	protected void init(VaadinRequest vaadinRequest)
	{
		Grid<Object> grid = new Grid<>();
		grid.setSizeFull();
		
		// make 100 hidable columns
		for (int fieldNo = 1; fieldNo <= 100; ++fieldNo)
		{
			grid.addColumn(e -> "?").setId("field" + fieldNo).setCaption("Field " + fieldNo).setHidable(true);
		}
		
		setContent(grid);
	}
	
	@WebServlet(urlPatterns = "/*", name = "MyUIServlet", asyncSupported = true)
	@VaadinServletConfiguration(ui = MyUI.class, productionMode = true)
	public static class MyUIServlet extends VaadinServlet
	{
	}
}
```

### Expected behavior
In IE11 the column list should allow users to toggle columns even after scrolling down the list.

### Actual behavior
In IE11 the column list jumps to the top after the user moves the mouse over the list, making it impossible to toggle columns further down the list.

**Comment by TatuLund** [comment]
Hi @geotim90 , can you check the issue https://github.com/vaadin/framework/issues/9569 and verify is your report about the same problem?

**Comment by geotim90** [comment]
Yes, @TatuLund , it looks like it describes the same problem. I couldn't find it before I created this issue. I guess you can mark this as a duplicate, but #9569 is labelled as "stale"...

**Comment by TatuLund** [comment]
There seems to be another duplicate here too: https://github.com/vaadin/framework/issues/7874

**Comment by geotim90** [comment]
Yes, but someone posted in #7874 that Google Chrome was affected back then, too...

**Comment by TatuLund** [comment]
It is possible that there has been bug in Chrome 55 as well, but it seems to be have been fixed since then.

**Comment by geotim90** [comment]
Is there any work-around we can use or something that can be done in Vaadin Framework 8 to make this work properly? IE11 is still officially supported, isn't it?

**Comment by TatuLund** [comment]
The workaround for this issue with IE11 is to create custom UI to select columns. It is possible to detect if the browser is IE11, if yes. Then have a button in UI which opens popup, where you can enabled / disable columns. It is possible to hide / unhide columns programmatically with Grid API.

**Comment by Ansku** [comment]
Closing as a duplicate, workaround copied to the oldest ticket.
